course-v2/layouts/partials/get_instructors.html

```diff
@@ -4,17 +4,15 @@
   {{ $url := (print (strings.TrimSuffix "/" $staticApiBaseUrl) "/instructors/" . "/index.json") }}
   {{ with resources.GetRemote $url }}
     {{ with .Err }}
-      {{ $errorMessage := printf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url .Err }}
-      {{ partial "sentry_capture_message.html" $errorMessage }}
```

Comment:
Two issues here:
1. This doesn't work at all. The intent of calling `{{ partial "sentry_capture_message.html }}` here is to include a <script> tag in the markup that pings sentry. This isn't clearly documented, but as far as I can tell, Hugo partials can EITHER return a value OR they can include markup. Partials cannot do both, which we were attempting to do here. So it didn't work at all.
2. Even if this did work, including a <script> tag is not really that great. It means we get the error when a user visits the page, not when the hugo build occurs. We can detect the error at build time, so let's do that.